earth_enterprise/src/installer/common.sh

```diff
@@ -250,6 +250,10 @@ create_links()
         ln -s $BASEINSTALLDIR_VAR/run $BASEINSTALLDIR_OPT/run
     fi
 
+    if [ ! -L "$BASEINSTALLDIR_OPT/share/doc" ]; then
```

Comment:
I'm a little confused by this - should we be checking if $BASEINSTALLDIR_OPT/gehttpd/htdocs/shared_assets/docs exists?